### DISCUSSION TITLE
Are Custom Attributes still supported?

### DISCUSSION THREAD

**Discussion by cameron-smtm** [tinybirdco/web-analytics-starter-kit#70] [Q&A]
The README suggests you can append a custom attribute to your data via a tag like `tb_customer_id="CUSTOMER_ID"` however I'm not sure if I'm doing it wrong but I haven't seen any of that tag data show up in the payload. Is this something still supported by flock script? Where can more info be found on the analytics script?

**Comment by radchukd** [comment]
👋 Yes, it is still supported. The attribute name must start with `tb_` and the value must be not `undefined`.

As for more info, only the info that we have in README. If you have more questions, I'd be happy to answer and then include it in README
